v22.2/cloud-storage-authentication.md

```diff
@@ -417,41 +488,125 @@ Service accounts in Google and Kubernetes refer to different resources. See [Goo
 
 To use assume role authentication, you will need at least two IAM roles:
 
-- An _identity service account_: the IAM service account you have associated with your Kubernetes service account.
+- An _identity service account_: the IAM service account associated with your Kubernetes service account.
 - An _operation service account_: the IAM service account to be assumed. This contains the permissions required to complete a CockroachDB operation.
 
-### Set up Google Cloud Storage workload identity
+For a step-by-step on how to set this up for {{ site.data.products.core }} and {{ site.data.products.dedicated }} clusters, see the following steps.
+
+### Step 1. Set up the Google Cloud workload identity
+
+The first step to setting up workload identity authentication is different for {{ site.data.products.core }} and {{ site.data.products.dedicated }} clusters. As you work through the step, ensure you use the relevant section for your cluster:
+
+- [{{ site.data.products.dedicated }}](#set-up-the-identity-service-account-for-dedicated-clusters)
+- [{{ site.data.products.core }}](#set-up-the-identity-service-account-for-self-hosted-clusters)
+
+#### Set up the identity service account for dedicated clusters
 
-Before completing the steps to run a bulk operation with assume role, it is necessary to create a identity service account for your Kubernetes service account to assume. Then, you must configure your CockroachDB pods to use the Kubernetes service account. You can complete all of these steps with Google's guide [Use Workload Identity](https://cloud.google.com/kubernetes-engine/docs/how-to/workload-identity).
+Each {{ site.data.products.dedicated }} cluster has a pre-configured IAM role for [Google Kubernetes Engine](https://cloud.google.com/kubernetes-engine/docs) service accounts that act as the cluster's identity. In addition, the clusters have a unique, built-in functional IAM role, which you can configure as a trusted identity by giving it access to an operation service account. This allows you to have the built-in IAM role assume another role (or roles as part of a [chain](#google-cloud-storage-role-chaining)).
+
+The built-in IAM identity roles are as follows, which become part of your Google identity service account name:
+
+IAM role name | Operation
+--------------+------------
+`crl-dr-store-user` | Backup, Restore, Import, Export
+`crl-cdc-sink-user` | Changefeeds
+`crl-kms-user` | Encrypted backups and restores
+
+Construct the service account name for your identity service account. You will need this to add to the operation role or intermediary role. The identity service account name follows this pattern (in this example the `crl-dr-store-user` role name is used for a backup):
+
+{% include_cached copy-clipboard.html %}
+~~~
+crl-dr-store-user-{cluster id suffix}@{account id}.iam.gserviceaccount.com
```

Comment:
nit: GCP lingo is "project ID", not "account ID" as with AWS.

Reply:
Good catch that should be `project ID`, as you say. I also added some clarification a couple of lines down since when you request the cluster's details from the API, the `account_id` in the output is what the user will be including as the GCP project ID.

Reply:
ty!